src/eos-google-chrome

```diff
@@ -94,6 +140,12 @@ class GoogleChromeLauncher:
         except OSError as e:
             exit_with_error("Could not launch Chrome: {}".format(repr(e)))
 
+        self._wait_for_installation()
```

Comment:
Here are my thoughts...  This should be a one-time only thing; other than if the download is interrupted or fails, it should continue to try again on subsequent logins until the installation is successful.  And, we should probably have the same retry after any reconnection to the network.  But, once successful, the gsetting should be cleared so that the automatic download never happens again.  One possible implementation...  as a last step of any installation of Chrome, check if the gsetting is set -- if so, set Chrome as the default browser and clear the gsetting.  In the meantime, if the user clicks on the Chrome icon and it is not installed, we should open the app center to the details page for Chrome, which would show any progress under way.  I don't think it is necessary to explicitly force the installation to start when clicking the desktop icon, as the installation should have started when the network connected.